### PR TITLE
Add missing Prow configs so that we can update to latest images

### PR DESCRIPTION
**Pull request by chaodaiG**
Part of #1693 

/assign @chizhg 
/cc @katharine

### CODE REVIEW

**Comment by chaodaiG** [comment]
@Katharine as fyi

File: ci/prow/deployments/deck.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ spec:
         - --tide-url=http://tide/
         - --spyglass
         - --oauth-url=/github-login
+        - --cookie-secret=/etc/cookie/secret
```
Comment (chizhg) [inline ci/prow/deployments/deck.yaml]: What is this cookie-secret? Is it also required to make this work?

Comment (chaodaiG) [reply ci/prow/deployments/deck.yaml]: Sorry for not making it clear, this is a by product

Comment (chaodaiG) [reply ci/prow/deployments/deck.yaml]: https://github.com/kubernetes/test-infra/blob/master/prow/ANNOUNCEMENTS.md
It was announced to be needed long time ago

Comment (chizhg) [reply ci/prow/deployments/deck.yaml]: I see, nice!

**Comment by chizhg** [comment]
/lgtm
/hold
In case @Katharine has extra comments.

**Pull request merged**